src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/SampleData.kt

```diff
@@ -124,6 +124,41 @@ class SampleData {
       )
     }
 
+    fun sampleActionPlan(
```

Comment:
please can we add this as one of the `Factory` classes in the test utils package? it would be good to keep all the test data in one place and i think those classes are better structured. (although not perfect, i've found that having to pass an entity manager to those classes is annoying for writing pure unit tests).

Reply:
will do